crowbar_framework/app/views/nodes/_node_groups.html.haml

```diff
@@ -7,16 +7,19 @@
     .col-xs-3{ "data-id" => group_name, "data-droppable" => @draggable.to_s, "data-group" => group_name }
       .panel.panel-default
         .panel-heading
-          %h2
-            = piechart_for group
```

Comment:
IMHO we should keep a generic indicator if one of the nodes is in a different state... maybe not this kind of piechart but still some indicator that makes it directly obvious that something is not green.